Artifacts.toml

```diff
@@ -1,5 +1,6 @@
 [meshcat]
-git-tree-sha1 = "15c3973b1084a231410e43e844c7714f1c41f163"
+git-tree-sha1 = "ed78da766b4ebd9b2179d1f34ffd74f212ae9bb7"
+
     [[meshcat.download]]
-    url = "https://github.com/rdeits/meshcat/tarball/978cb8f519f9bb540e94b7f97a39ada4d7916b7c"
-    sha256 = "b3a5343bd0fcaabff4fcb73c0951a32d887d1cf0c69a64b52ea957704d4d1e0a"
```

Comment:
Self-reminder: update the artifact so it points back to `meshcat-dev` (instead of `ferrolho`) once https://github.com/meshcat-dev/meshcat/pull/180 is merged.